configs/components/echam/echam.yaml

```diff
@@ -73,7 +73,7 @@ compile_infos:
             branch: "wiso"
             clean_command: rm -rf src/echam/bin; rm -rf bin; make clean
             comp_command: ./config/createMakefiles.pl; autoreconf -i --force; mkdir -p src/.deps yaxt/src/.deps yaxt/tests/.deps; ./configure $configure_opts --with-fortran=intel INSTALL='/usr/bin/install -p'; make -j `nproc --all`; make install -j `nproc --all`; mkdir -p src/echam/bin; cp  bin/echam6 src/echam/bin/echam6
-            git-repository: "https://gitlab.awi.de/pgierz/echam6.git"
+            git-repository: "https://gitlab.awi.de/paleodyn/Models/echam6.git"
```

Comment:
Are WISO people working on this repo already? Otherwise, we need to let them know that they should be changing repos.